app/services/PenaltiesService.scala

```diff
@@ -73,16 +73,16 @@ class PenaltiesService @Inject()(connector: PenaltiesConnector) {
       allLPPs => {
         val allAdditionalPoints = allLPPs.filter(_.`type` == PenaltyTypeEnum.Additional)
         val allFinancialPoints = allLPPs.filter(_.`type` == PenaltyTypeEnum.Financial)
-        val estimatedLPPs = allAdditionalPoints.map(_.financial.amountDue).foldRight(BigDecimal(0))(_ + _)
-        val crystallisedLPPs = allFinancialPoints.map(_.financial.amountDue).foldRight(BigDecimal(0))(_ + _)
```

Comment:
Can the 'estimatedLPPs' and 'crystallisedLPPs' be renamed, looks like they are actually getting the outstandingAmountDue from Additional and Financial

Reply:
They are getting the values from the same place but for different types of penalties - the estimated LPP's comes from the additional penalties that the user may have whereas the crystallised LPP's comes from all the LPP's for that period and have been finalised.